.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org/
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
nit: should this also comply with `insert_final_newline = true`?

Reply:
good one, will add